eva_submission/eload_validation.py

```diff
@@ -117,12 +126,23 @@ def parse_vcf_check_report(self, vcf_check_report):
                         error_list.append(line.strip())
         return valid, error_list, error_count, warning_count
 
+    def _generate_csv_mappings(self):
+        vcf_files_csv = os.path.join(self.eload_dir, 'vcf_files.csv')
+        with open(vcf_files_csv, 'w', newline='') as file:
+            writer = csv.writer(file)
+            writer.writerow(['vcf', 'fasta', 'report'])
+            for analysis_alias in self.eload_cfg.query('submission', 'analyses'):
```

Comment:
I would find it a bit more readable to first get `analysis_dict = self.eload_cfg.query('submission', 'analyses')` and then use the dict to get the relevant attributes.